docs/docs/gatsby-cli.md

```diff
@@ -23,38 +22,38 @@ You can also use the `package.json` script variant of these commands, typically
 }
 ```
 
-## API commands
+## एपीआई कमांड
 
-### `new`
+### `नया`
 
 ```
 gatsby new [<site-name> [<starter-url>]]
 ```
 
-#### Arguments
+#### तर्क
 
-| Argument    | Description                                                                                                                                                                                                     |
+| तर्क    | विवरण                                                                                                                                                                                                     |
 | ----------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| site-name   | Your Gatsby site name, which is also used to create a project directory.                                                                                                                                        |
-| starter-url | A Gatsby starter URL or local file path. Defaults to [gatsby-starter-default](https://github.com/gatsbyjs/gatsby-starter-default); see the [Gatsby starters](/docs/gatsby-starters/) docs for more information. |
+| site-name   | आपका gatsby साइट का नाम, जिसका उपयोग प्रोजेक्ट डायरेक्टरी बनाने के लिए भी किया जाता है।                                                                                                                                        |
+| starter-url | एक gatsby स्टार्टर URL या स्थानीय फ़ाइल पथ। के लिए चूक [gatsby-starter-default](https://github.com/gatsbyjs/gatsby-starter-default); देखें [Gatsby starters](/docs/gatsby-starters/) अधिक जानकारी के लिए डॉक्स। |
```

Comment:
`gatsby` => `Gatsby`

Reply:
you have missed this

Reply:
You have missed this again 😛

Reply:
you mean at line 38: gatsby-starter-default => Gatsby-starter-default
Would appreciate if can add line no as well going forward :)

Reply:
`एक gatsby स्टार्टर` => `एक Gatsby स्टार्टर`
line 38.

Github review comments can tell you the line number. Comments gets added right below the line.
Basically, whatever the last line is visible above the feedback, comment feedback is talking about that line.
If you still have difficulties to find the line number, I can include the line numbers going forward. Let me know whatever works for you